tests/test-logging-monitoring/test-lambda-dashboard-requirements.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type PrometheusSeries struct {
+	Status string              `json:status`
+	Data   []map[string]string `json:data`
+}
+
+func checkMetricsAndlabels(metric string, labels ...string) error {
+	url := prometheusURL + "/api/v1/series"
+	url += "?match[]=" + metric
+
+	for _, l := range labels {
+		u := fmt.Sprintf("%s{%s=~\"..*\"}", url, l)
+		s := PrometheusSeries{}
+
+		resp, err := http.Get(u)
+		if err != nil {
+			return fmt.Errorf("Call to prometheus failed: %v", err)
+		}
+
+		defer resp.Body.Close()
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		json.Unmarshal(bodyBytes, &s)
+
+		if s.Status != "success" {
+			return fmt.Errorf("Call to prometheus failed with response: %v", s)
+		}
+
+		if len(s.Data) < 1 {
+			return fmt.Errorf("Metric or Lable not found: %s, %s", l, metric)
```

Comment:
typo: `label` also the order is wrong it should be: `return fmt.Errorf("Metric or Lable not found: %s, %s", metric, l)`

Reply:
fixed